### PR TITLE
Update plot_time_series and test file

### LINKED ISSUE #18: Write function and create tests for plot_time_series()

### PR DESCRIPTION
**Pull request by SiqiTao**

### CODE REVIEW

File: src/CovidTracker/plot_time_series.py [hunk 1]
```diff
@@ -14,4 +34,56 @@ def plot_ts(df):
     Examples
     ----------
     >>> plot_ts(covid_df)
```
Comment (jessie14) [inline src/CovidTracker/plot_time_series.py]: example function call missing metric input 
ie should say something like plot_ts(covid_df, 'cases')

File: src/CovidTracker/plot_time_series.py [hunk 2]
```diff
@@ -1,11 +1,31 @@
-def plot_ts(df):
+from get_covid_data import get_covid_data 
+import matplotlib.pyplot as plt
```
Comment (jessie14) [inline src/CovidTracker/plot_time_series.py]: is matplotlib used in your function anywhere?

File: tests/test_timeseries.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
+from CovidTracker.get_covid_data import get_covid_data
+from CovidTracker.plot_time_series import plot_ts
+from pytest import raises
+import pandas as pd
+import matplotlib as plt
+import pytest
+
+
+def test_plot_ts():
```
Comment (jessie14) [inline tests/test_timeseries.py:8]: Should test more branches of the function. For example, if start and end dates are given, if given start date is earlier than min date in covid_df, etc

File: src/CovidTracker/plot_time_series.py (hunk 1 above)
Comment (SiqiTao) [reply src/CovidTracker/plot_time_series.py]: fixed, thanks!

File: src/CovidTracker/plot_time_series.py (hunk 2 above)
Comment (SiqiTao) [reply src/CovidTracker/plot_time_series.py]: lol no, i turned out deciding to use altair, good catch!

**Pull request merged**